pkg/object/object.go

```diff
@@ -45,9 +48,27 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func (s *Service) Validate() error {
+	// validate service name, like it cannot have underscores, etc.
+
+	// validate containers
+
+	// validate replicas
+	if s.Replicas != nil && *s.Replicas < 0 {
```

Comment:
Letes move on, and go with this as it is.
We can figure out how we are going to do all validations in another PR.

https://github.com/redhat-developer/opencompose/issues/92